tests/functional/conftest.py

```diff
@@ -34,6 +34,11 @@ def gitlab_version(gl) -> GitlabVersion:
     return GitlabVersion(major=major, minor=minor, patch=patch, revision=revision)
 
 
+@pytest.fixture(scope="session")
+def docker_compose_command():
+    return "docker compose"
```

Comment:
So that we use can the go-based v2 compose:
https://github.com/avast/pytest-docker#docker_compose_command

Reply:
As a note this breaks my current `podman` based way of running the tests. But can work on that later.